outtake_time: 0.1 # seconds, how long to run outtake for before intake